velox/functions/prestosql/ArrayAndMapMatch.cpp

```diff
@@ -25,7 +25,11 @@ namespace facebook::velox::functions {
 namespace {
 
 // @tparam TContainer Either ArrayVector or MapVector.
-template <typename TContainer, bool initialValue, bool earlyReturn>
+template <
+    typename TContainer,
+    bool initialValue,
+    bool earlyReturn,
+    bool followNull = true>
```

Comment:
Hi, we may need to move the implementation to the lib folder to support the Spark-compatible behavior, thanks.